src/VisualStudio/Xaml/Impl/Implementation/XamlProjectService_IVsRunningDocTableEvents3.cs

```diff
@@ -4,7 +4,9 @@
 
 #nullable disable
 
+using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
 using Microsoft.VisualStudio.Shell.Interop;
```

Comment:
Why does removing code require new usings?

Reply:
Oops it shouldn't. Don't remember how did they get added. I will clean it up.

---
In reply to: [606385567](https://github.com/dotnet/roslyn/pull/52369#discussion_r606385567) [](ancestors = 606385567)